framework/components/ADataTable/ADataTable.scss

```diff
@@ -1,5 +1,29 @@
 @import "../../styles";
 
+$hidden-table-col-width: 30px;
+
+@include theme(a-data-table--expandable) using ($theme) {
```

Comment:
Like above, theme should be on `a-data-table` and contents of this should be wrapped in `&--expandable { ... }`